frontend/src/components/place/SearchPlace.vue

```diff
@@ -57,49 +55,83 @@ export default {
             isMapLoaded: false,
             isPlacesLoaded: false,
             map: {},
+            userCoordinates: {
+                lat: '',
+                lng: ''
+            },
         };
     },
     created() {
         this.$store.dispatch('place/fetchPlaces')
-            .then(() => this.isPlacesLoaded = true);
+            .then(() => {
+                this.isPlacesLoaded = true;
+                if(this.isMapLoaded){
+                    this.updateMap(this.places);
+                    markerManager.fitMarkersOnMap();
+                }
+            });
     },
     methods: {
         mapInitialized(map) {
             this.map = map;
             LocationService.getUserLocationData()
                 .then(coordinates => {
-                    this.jumpTo(coordinates);
+                    this.userCoordinates.lat = coordinates.lat;
+                    this.userCoordinates.lng = coordinates.lng;
                 });
         },
         mapLoaded(map) {
             markerManager = new MarkerService(map);
             this.isMapLoaded = true;
+            if(this.isPlacesLoaded){
+                this.updateMap(this.places);
+                markerManager.fitMarkersOnMap();
+            };
         },
-        jumpTo (coordinates) {
+        jumpTo(coordinates) {
             this.map.jumpTo({
                 center: coordinates,
             });
         },
-        updateMap(places){
+        updateMap(places) {
             markerManager.setMarkers(...places);
+        },
+        createUserMarker(){
+            return {
+                id           : 0,
+                latitude     : this.coordinates.lat,
+                longitude    : this.coordinates.lng,
+                localization : {
+                    0: {
+                        description: 'Your position',
+                        language: 'en',
+                        name: 'Your position',
+                    }
+                },
+                photoUrl: this.user.avatar_url || 'https://www.microsoft.com/en-us/research/uploads/prod/2018/02/ProfilePhoto.jpg',
```

Comment:
Who is this? Could you remove it?

https://www.microsoft.com/en-us/research/uploads/prod/2018/02/ProfilePhoto.jpg